scalapb/scalapb_2.13/src/test/resources/com/linecorp/armeria/server/protobuf/ScalaPbNamedTypeInfoProviderTest_specification.json5

```diff
@@ -254,41 +255,7 @@
                           "location": "UNSPECIFIED",
                           "requirement": "OPTIONAL",
                           "typeSignature": "armeria.protobuf.testing.SimpleOneof",
-                          "childFieldInfos": [
```

Comment:
Just double checking, are we removing `childFieldInfos` from the return type signatures for all types? What is this about?

Reply:
It's a bug. There's a recursion so I removed it. You can find `armeria.protobuf.testing.SimpleOneof"` in one two levels up.